orchestration/pyproject.toml

```diff
@@ -41,6 +41,7 @@ check = "hca_manage.check:run"
 dataset = "hca_manage.dataset:run"
 snapshot = "hca_manage.snapshot:run"
 soft_delete = "hca_manage.soft_delete:run"
+job = "hca_manage.job:fetch_job_info"
```

Comment:
This was annoying me, added as a command here so I don't have to always do a `python hca_manage/job.py blahblah`